src/components/friends/UserCard.tsx

```diff
@@ -40,17 +46,47 @@ const UserCard: FC<UserCardProps> = (props) => {
   );
 
   return (
-    <div className="col-4" key={id}>
-      <div>
-        <div>{userName}</div>
-        <div>{displayName}</div>
-      </div>
-      <button onClick={handleFollowClick}>
-        {isFollowed ? "Unfollow" : "Follow"}
-      </button>
-      {onFollowerRemove && (
-        <button onClick={handleRemoveFollower}>Remove</button>
-      )}
+    <div className="col-3" key={id}>
+      <Card
```

Comment:
use styled-components. For example.
const StyledCard = styled(Card)`
   ....
`.
styled-components can be for everything as well